client/src/components/Home.js

```diff
@@ -0,0 +1,12 @@
+import Map from './Map'
+
+function Home(props) {
+    return(
+        <div id="home-page">
+            <Map />
+       
+        </div>
+    )
+}
+
+export default Home
```

Comment:
We could probably just import your `<Map/>` directly into where you want to use it. This intermediate step isn't doing much other than wrapping your map in a `div`